test_finder.sh

```diff
@@ -18,6 +18,7 @@ OSD_COMPONENT_TEST_MAP=( "OpenSearch-Dashboards:opensearch-dashboards"
                          "securityDashboards:security"
                          "notificationsDashboards:notifications-dashboards"
                          "customImportMapDashboards:custom-import-map-dashboards"
+                         "searchRelevanceDashboards:search-relevance-dashboards"
```

Comment:
I use [searchRelevanceDashboards](https://github.com/opensearch-project/dashboards-search-relevance/blob/ede673f031f9d431589a97a33ceaa7a5741ed397/opensearch_dashboards.json#L2) as left part and use `cypress/integration/plugins/search-relevance-dashboards/` folder name as right part

@tianleh Can you help me verify if it is correct usage? Thanks!

Reply:
This is correct. The left side is the name in infra's build repo https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.4.0/opensearch-dashboards-2.4.0.yml#L55 The right side is the name of the file folder.